pandasgui/widgets/plotly_viewer.py

```diff
@@ -33,6 +34,9 @@
 
         app.__init__(sys.argv + ["--ignore-gpu-blacklist", "--enable-gpu-rasterization"])
 
+# Available symbol names for a given version of Plotly
+_extended_symbols = SymbolValidator().values[0::2][1::3]
+plotly_markers = [symbol for symbol in _extended_symbols if symbol[-3:] != "dot"]
```

Comment:
Those dotted symbols can't be easily distinguished, hence why I'm filtering them